rotkehlchen/chain/arbitrum_one/modules/gmx/balances.py

```diff
@@ -83,10 +85,9 @@ def query_balances(self) -> 'BalancesType':
         deposit (only its USD value) and a delta value that is how much the position has earned
         (also in USD). We sum collateral value + delta to get the current value of the position.
         """
-        balances: BalancesType = defaultdict(lambda: defaultdict(Balance))
```

Comment:
you need to adjust the docstring here to explain that norhing is returned. And explain since it returns nothing what happens.

Seems to modify the passed balances argument?

Reply:
As per the suggestion below now it returns it